Packs/Base/Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.py

```diff
@@ -1,16 +1,21 @@
-from CommonServerPython import *
 import base64
+import copy
+
+from CommonServerPython import *
 
 
 def main():
-    get_incidents_by_query_args = dict(demisto.args())
-    get_incidents_by_query_args['NonEmptyFields'] = demisto.args()['tagField']
+    d_args = dict(demisto.args())
+    for arg in ['tagField', 'emailbody', 'emailbodyhtml', 'emailsubject', 'timeField']:
+        d_args[arg] = d_args[arg].replace('incident.', '')
+    get_incidents_by_query_args = copy.deepcopy(d_args)
+    get_incidents_by_query_args['NonEmptyFields'] = d_args['tagField']
```

Comment:
please add also another thing there - set the `populateFields` arg for this automation, to have all the fields we need (which all the fields above in the iteration: ['tagField', 'emailbody', 'emailbodyhtml', 'emailsubject', 'timeField']

Reply:
did it one a seperate pr (not yet completed): https://github.com/demisto/content/pull/6811/files